src/configuration.ts

```diff
@@ -20,9 +20,6 @@ export class Configuration {
 	project = '';
```

Comment:
Nit: unrelated but we can safely remove sdkKey because it seems unused.

Reply:
Jut removed it, agreed it looks like we're grabbing the SDK key directly out of the environments response so this should be unneeded at this point. I believe it was left in there for legacy reasons but not 100% sure on that.